Sources/SourceControl/GitRepository.swift

```diff
@@ -83,7 +83,7 @@ public struct GitRepositoryProvider: RepositoryProvider {
         precondition(!localFileSystem.exists(path))
 
         // FIXME: Ideally we should pass `--progress` here and report status regularly.  We currently don't have callbacks for that.
-        try self.callGit("clone", "--mirror", repository.url, path.pathString,
+        try self.callGit("clone", "-c", "core.symlinks=true", "--mirror", repository.url, path.pathString,
```

Comment:
please add a code comment on why this was added for future maintainers!

Reply:
(here as well as in the other spots)

Reply:
Done.